README.md

```diff
@@ -1,33 +1,147 @@
+```
+
+## Quick Start ⚡
+
+<details>
+<summary><b>Create a chatbot with prompt flow</b></summary>
+ It creates a new <b>flow folder</b> named my_chatbot and generates the necessary flow files within it. The --flow argument is the path to the flow folder.
+</details>
+
+Initiate a prompt flow from a chat template:
+
+```sh
+pf flow init --flow ./my_chatbot --type chat
+```
+
+<details>
+<summary><b>Setup a connection for your API key</b></summary>
+Navigate to the `my_chatbot` folder, you can find a yaml file named `openai.yaml` file, which is the definition of the connection to store your Open AI key.
+</details>
+
+> <details>
+> <summary>For Azure OpenAI key, establish the connection by running:</summary>
+>
+> ```sh
+> # Override keys and endpoint with --set to avoid yaml file changes
+> pf connection create --file ./my_chatbot/azure_openai.yaml --set api_key=<your_api_key> api_base=<your_api_endpoint>
+> ```
+> </details>
+
+For OpenAI key, establish the connection by running:
+
+
+```sh
+# Override keys with --set to avoid yaml file changes
+pf connection create --file ./my_chatbot/openai.yaml --set api_key=<your_api_key>
+```
+
+<details>
+<summary><b>Chat with your flow</b></summary>
+In the `my_chatbot` folder, there's a `flow.dag.yaml` file that outlines the flow, including inputs/outputs, tools, nodes, etc. Note we're using the connection named `open_ai_connection` in the `chat` node.
+</details>
+
+> <details>
+> <summary>For Azure Open AI users, modify this file accordingly.</summary>
+> Replace the 'node:' section with following content and specify the 'deployment_name' to the model deployment you'd like to use.
+>
+> ```yaml
+> nodes:
+> - name: chat
+>   type: llm
+>   source:
+>     type: code
+>     path: chat.jinja2
+>   inputs:
+>     deployment_name: <your_azure_open_ai_deployment_name>
+>     max_tokens: '256'
+>     temperature: '0.7'
+>     chat_history: ${inputs.chat_history}
+>     question: ${inputs.question}
+>   api: chat
+>   connection: azure_open_ai_connection
+> ```
+>  </details>
+
+For OpenAI users, interact with your chatbot by running: (press `Ctrl + C` to end the session)
+
+```sh
+pf flow test --flow ./my_chatbot --interactive
+```
+
+Then you will see the chatbot in action:
+![interactive](examples/tutorials/quick-start/media/interactive.gif)
+
+Next Step! Continue with the **Tutorial**  👇 section to delve deeper into Prompt flow.
+
+## Tutorial 🏃‍♂️
+
+Prompt Flow is a tool designed to **facilitate high quality LLM-native apps to production**, the development process in prompt flow follows these steps: develop a flow, improve the flow quality, deploy the flow to production.
+
+### Develop your own LLM apps
+
+[Getting Started with Prompt Flow](https://microsoft.github.io/promptflow/how-to-guides/quick-start.html): A step by step guidance to invoke your first flow run.
+
+#### VS Code Extension<img src="examples/tutorials/quick-start/media/logo_pf.png" alt="logo" width="25"/> 
```

Comment:
We don't want this video on README page, it should be included in "Get Started"

Reply:
moved to "Get Started"，replaced here with screenshot to attract user

Reply:
Let's target to add the easy start sample this time. Adding git is good, but the question in the sample is not proper... not smart, no intelligence, not attractive at all.